pkg/clusterstack/factory.go

```diff
@@ -21,7 +21,12 @@ type ImageFetcher interface {
 }
 
 type ImageRelocator interface {
-	Relocate(writer io.Writer, image v1.Image, dest string, tlsCfg registry.TLSConfig) (string, error)
+	Relocate(image v1.Image, dest string, writer io.Writer, tlsCfg registry.TLSConfig) (string, error)
```

Comment:
Why the signature arrange?

Reply:
It made more sense when reading.. What is being relocated? the image (and not the writer). They are support parameters.
partly inspired by [this](https://github.com/golang/go/wiki/CodeReviewComments#contexts) a bit

Reply:
marking as resolved. please reopen if not ok